pallets/author-filter/src/lib.rs

```diff
@@ -67,6 +67,9 @@ pub mod pallet {
 	impl<T: Config> author_inherent::CanAuthor<T::AccountId> for Pallet<T> {
 		fn can_author(account: &T::AccountId) -> bool {
 			let mut staked: Vec<T::AccountId> = stake::Module::<T>::validators();
+			if staked.len() == 1 {
+				return staked.contains(account);
+			}
```

Comment:
Did you find that the pallet was not selecting the single staked author as eligible before adding this? I thought it was already working with a single staked validator because I used the [`mul_ceil` method](https://crates.parity.io/sp_arithmetic/per_things/trait.PerThing.html#method.mul_ceil)

Reply:
Don't the passing integration tests confirm that? Gerald is the only staked candidate in the dev config.

Reply:
When testing with 1 validator I'm getting a `division by zero` panic somewhere in the loop